.github/workflows/job-node-publish.yml

```diff
@@ -64,5 +70,6 @@ jobs:
         git tag -a ${{inputs.version}} -m "[no ci] Release ${{inputs.version}}"
     - name: Push
       run: |
+        npm version ${{inputs.version}} -m "[no ci] Release ${{inputs.version}}"
```

Comment:
why do we need this change?
the above code will do the commit - is the work dir there correct?

Reply:
we have **2 pakege.jsons**, so firstly we update the version of the generated by angular one in line 57 (this one in outside of git, so nothing happens with commit), because it contains the original version and once in package registy we need to have it with proper version, what happens here line 73 we update the original package.json which goes with a new version to git

Reply:
of course in case the working dir is the same in both actions we just update twice the same wile with the sane version